presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -251,12 +251,13 @@ else if (type instanceof MapType) {
                             e -> convertToRowValue(((MapType) type).getValueType(), e.getValue())));
         }
         else if (type instanceof RowType) {
-            Map<String, Object> data = (Map<String, Object>) value;
-            RowType rowType = (RowType) type;
-
-            return rowType.getFields().stream()
-                    .map(field -> convertToRowValue(field.getType(), data.get(field.getName().get())))
-                    .collect(toList());
+            List<Object> data = new ArrayList<>(((Map<String, Object>) value).values());
```

Comment:
Can you check if iteration order of this map is deterministic? If so, add a check here that map if of that type.

Reply:
I don't follow what do we want to check here?